sys/kern/uipc_mqueue.c

```diff
@@ -2012,7 +2012,7 @@ kern_kmq_open(struct thread *td, const char *upath, int flags, mode_t mode,
 	struct pwddesc *pdp;
 	struct file *fp;
 	struct mqueue *mq;
-	int fd, error, len, cmode;
+	int fd, error, len, cmode, slash;
```

Comment:
Instead of adding the slash index, could you rename path to e.g. pathbuf, and have char *path pointing to the slash-less string?

Reply:
> Instead of adding the slash index, could you rename path to e.g. pathbuf, and have char *path pointing to the slash-less string?

Fixed.